root/etc/e-smith/templates/etc/opt/rh/rh-php72/php-fpm.d/000-nextcloud.conf/10base

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Please transform this template to a simple configuration file (no replace?).
Also remove all properties from the db.

The admin will be able to tune the configuration just editing the file.

Reply:
ok

Reply:
it is not possible we need it to create the relevant date.timezone or we let nextcloud as is with default UTC timezone like before

Reply:
ok done, we just increase memory_limit